### PR TITLE
Set up initial version

### LINKED ISSUE #1: Set up initial version

### PR DESCRIPTION
**Pull request by jdeflander**
closes #1

### CODE REVIEW

**Comment by TomGijselinck** [comment]
I'm not sure if this PR is ready for review because it's still in draft. Does it need some additional work before changing the draft to normal?

**Comment by jdeflander** [comment]
> I'm not sure if this PR is ready for review because it's still in draft. Does it need some additional work before changing the draft to normal?

I was thinking about adding unit tests, but I don't think it makes a lot of sense in this case. I'm also considering adding command-line flags to set some variables that are currently hard-coded. Except for that part, everything is ready for review though 😄

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier",
+    "prettier/@typescript-eslint"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"]
+}
```
Comment (TomGijselinck) [inline .eslintrc.json:10]: Waarom eslint en geen tslint?

**Review by TomGijselinck — DISMISSED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (rikbw) [reply .eslintrc.json:10]: Das deprecated in favor of eslint

Comment (TomGijselinck) [reply .eslintrc.json:10]: Moeten we dat dan best ook niet aanpassen bij insights?

**Review by rikbw — COMMENTED** [review]
LGTM

File: .eslintrc.json (hunk 1 above)
Comment (jdeflander) [reply .eslintrc.json:10]: > Moeten we dat dan best ook niet aanpassen bij insights?

Jep, best wel. Ik kan er een issue voor aanmaken als je wil?

Comment (TomGijselinck) [reply .eslintrc.json:10]: Lijkt me wel de moeite. 👍

**Review by TomGijselinck — DISMISSED** [review]

**Review by rikbw — APPROVED** [review]

**Pull request merged**